test/src/test/java/jenkins/agents/WebSocketAgentsTest.java

```diff
@@ -99,9 +100,10 @@ public void smokes() throws Exception {
             ).stdout(System.out).start());
             r.waitOnline(s);
             assertEquals("response", s.getChannel().call(new DummyTask()));
+            assertNotNull(s.getChannel().call(new FatTask()));
             FreeStyleProject p = r.createFreeStyleProject();
             p.setAssignedNode(s);
-            p.getBuildersList().add(Functions.isWindows() ? new BatchFile("echo hello") : new Shell("echo hello"));
+            p.getBuildersList().add(Functions.isWindows() ? new BatchFile("echo hello") : new Shell("dd if=/dev/random count=1024 bs=200"));
```

Comment:
You did not include all of #4635, specifically: https://github.com/jenkinsci/jenkins/pull/4635/commits/27626ccbd0d388d635bb73c7f444f7195045afae

Reply:
Good catch, reverting the isolated backport and pulling the whole #4635.